app/Http/Controllers/Legacy/Web/CampaignIdsController.php

```diff
@@ -85,7 +85,6 @@ public function show(Campaign $campaign)
         return view('pages.campaign_id_single')
             ->with('state', [
                 'campaign' => $campaign,
-                'actions' => $campaign->actions,
```

Comment:
Yes it does! But I was passing it as a prop before and we actually want it to be in the state (since when we delete the action, we want to update the state to remove it from the page).

Reply:
Ooo I see we use the API to grab the actions now?

Reply:
Exactly!